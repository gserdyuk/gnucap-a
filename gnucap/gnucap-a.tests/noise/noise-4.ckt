noise ckt with complex interacton
v1   1  0  dc 1 ac 1
R1   1  0  100.
R2   1  2  10
R3   2  0  100



.store ac vm(1) vp(1) temp(0)
.print ac vm(1) vp(1) temp(0)
.ac dec 10 10 100
.measure maxm = max("vm(1)")


.store noise inoise onoise
.print noise  inoise onoise 
.noise v(2) v1 dec 10 10 100

.measure maxn=max( "onoise(0)" )
* it is possible to operate with onoise and inoise with zero index

* values onoise_total an inoise_total are injeced 
* as measurements and can be addressed afterwards:

.measure r=eval(onoise_total/inoise_total)
.measure r2=eval(onoise_total*maxm)


.end
