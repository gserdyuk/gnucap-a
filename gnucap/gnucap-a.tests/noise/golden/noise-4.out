Gnucap gnucap-arails  2009.12.07 RCS 26.136 master 2010-09-25  bmm_semi_hsp, bm_model fix, back to bmm_semi
The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2009, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
ac: already installed, replacing
stashing as ac:0
noise ckt with complex interacton 
#Freq       vm(1)      vp(1)      temp(0)   
 10.        1.         0.         0.        
 12.589     1.         0.         0.        
 15.849     1.         0.         0.        
 19.953     1.         0.         0.        
 25.119     1.         0.         0.        
 31.623     1.         0.         0.        
 39.811     1.         0.         0.        
 50.119     1.         0.         0.        
 63.096     1.         0.         0.        
 79.433     1.         0.         0.        
 100.       1.         0.         0.        
maxm= 0.9999998
#Freq       inoise(0)  onoise(0) 
 10.        186.4E-21  154.E-21  
 12.589     186.4E-21  154.E-21  
 15.849     186.4E-21  154.E-21  
 19.953     186.4E-21  154.E-21  
 25.119     186.4E-21  154.E-21  
 31.623     186.4E-21  154.E-21  
 39.811     186.4E-21  154.E-21  
 50.119     186.4E-21  154.E-21  
 63.096     186.4E-21  154.E-21  
 79.433     186.4E-21  154.E-21  
 100.       186.4E-21  154.E-21  
inoise_total= 16.7746E-18
onoise_total= 13.8633E-18
maxn= 154.037E-21
r= 0.8264459
r2= 13.8633E-18
